SimPPS/RPDigiProducer/plugins/RPDigiProducer.cc

```diff
@@ -53,7 +52,7 @@ namespace CLHEP {
 class RPDigiProducer : public edm::EDProducer {
```

Comment:
This module needs to be migrated to some thread-efficient base class.

Is this module run in `runTheMatrix`? It is included in a list of modules being run that I extracted in last spring, but I don't see it being flagged as one preventing efficient use of concurrent lumis (that legacy EDModules imply) in https://github.com/cms-sw/cmssw/issues/25090.